backend/main.py

```
@@ -1,52 +1,49 @@
 
 
 
-from fastapi import FastAPI, Depends, HTTPException,Header
+from fastapi import FastAPI, Depends, HTTPException,Header, APIRouter, BackgroundTasks
 from fastapi.security import OAuth2PasswordBearer
 from sqlalchemy.orm import Session
 from database import SessionLocal, engine
 from models import User
-from schemas import UserCreate, Token, LoginRequest, UserUpdate
-from utils import auth_utils, create_access_token, verify_token, create_email_verification_token, send_verification_email
+from schemas import UserCreate, Token, LoginRequest, UserUpdate, OrderOut, OrderCreate, PayBillRequest, ForgotPasswordRequest, ResetPasswordRequest
```

Ruff (0.8.2)

`schemas.OrderOut` imported but unused

Remove unused import

(F401)